src/app/content/search/components/SearchResultsSidebar/styled.tsx

```diff
@@ -0,0 +1,208 @@
+  margin-top: 0;
+  padding: 0;
+  position: sticky;
+  width: ${searchResultsBarDesktopWidth}rem;
+  background-color: ${searchResultsBarVariables.backgroundColor};
+  box-shadow: 0.2rem 0 0.2rem 0 rgba(0, 0, 0, 0.1);
+  z-index: 2;
+  height: calc(100vh - ${navDesktopHeight + bookBannerDesktopMiniHeight + toolbarDesktopHeight}rem);
+
+  ${theme.breakpoints.mobile(css`
+    width: 100%;
+    margin-top: 0;
+    top: ${searchSidebarTopOffset}rem;
+    padding: 0;
+  `)}
+
+  > ${NavOl} {
+    ::before {
+      display: none;
+    }
+    margin: 0;
+    padding: 0;
+  }
+
+`;
+
+// tslint:disable-next-line:variable-name
+export const SearchQuery = styled.div`
+  ${textRegularStyle}
+  display: flex;
+  align-items: center;
+  justify-content: end;
+  min-height: 4rem;
+
+  strong {
+    padding-left: 0.4rem;
+  }
+`;
+
+// tslint:disable-next-line:variable-name
+export const SearchQueryWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  background: ${theme.color.neutral.base};
+  padding: 1rem 0;
+`;
+
+// tslint:disable-next-line:variable-name
+export const SearchBarSummary = styled(Summary)`
+    min-height: 3.8rem;
+    display: flex;
+    align-items: center;
+    background: ${searchResultsBarVariables.backgroundColor};
+    border-top: solid 0.1rem #d5d5d5;
+    padding-left: ${searchResultsBarVariables.mainPaddingDesktop}rem;
+`;
+
+// tslint:disable-next-line:variable-name
+export const SearchResultsLink = styled.div`
+  ${labelStyle}
+  width: 100%;
+`;
+
+// tslint:disable-next-line:variable-name
+export const SectionContentPreview = styled(ContentLinkComponent)`
+  ${labelStyle}
+  margin-left: ${searchResultsBarVariables.mainPaddingDesktop + iconSize + 2.3}rem;
+  min-height: 3.7rem;
+  align-items: center;
+  padding-right: 1.6rem;
+  padding: 1rem 0;
+  max-width: 35.3rem;
+  display: block;
+  text-decoration: none;
+
+  :not(:last-child) {
+    border-bottom: solid 0.1rem ${searchResultsBarVariables.backgroundColor};
+  }
+
+  em {
+    font-weight: bold;
+  }
+`;
+
+// tslint:disable-next-line:variable-name
+export const LinkWrapper = styled.div`
+    min-height: 3.4rem;
+    display: flex;
+    align-items: center;
+    padding-left: ${searchResultsBarVariables.mainPaddingDesktop + iconSize}rem;
+    padding-top: 1rem;
+    padding-bottom: 0.6rem;
+`;
+
+// tslint:disable-next-line:variable-name
+export const DetailsOl = styled.ol`
+  padding: 0;
+`;
+
+// tslint:disable-next-line:variable-name
+export const NavItem = styled.li`
+  background: ${theme.color.primary.gray.foreground};
+  :not(:last-child) {
+    border-bottom: solid 0.2rem ${searchResultsBarVariables.backgroundColor};
+  }
+`;
+
+// tslint:disable-next-line:variable-name
+export const SearchQueryAlignment = styled.div`
+  max-width: 26.5rem;
+  text-align: justify;
+  /*textRegularLineHeight is the close icon height*/
+  margin-top: calc(7rem - ${textRegularLineHeight}rem);
```

Comment:
why do you have to pull this up by the height of the icon? maybe the icon should be absolutely positioned? or this should be a row? i forget what it looks like '